Tasks/DownloadBuildArtifacts/task.loc.json

```diff
@@ -10,7 +10,7 @@
   "version": {
     "Major": 0,
     "Minor": 1,
```

Comment:
should we also switch this task to use minor=milestone (126)?

Reply:
not sure about impact, will take later.